test/test_respondent_controller.py

```diff
@@ -101,6 +101,26 @@ def test_get_respondent_by_email_returns_correct_representation(self):
         self.assertEqual(response['sampleUnitType'], self.mock_respondent['sampleUnitType'])
         self.assertEqual(response['telephone'], self.mock_respondent['telephone'])
 
+    def test_change_respondent_details_success(self):
+        self.populate_with_respondent()
+        payload = {
+            "respondent_id": "438df969-7c9c-4cd4-a89b-ac88cf0bfdf3",
+            "firstName": "John",
+            "lastName": "Snow",
+            "telephone": "07837230942"
+        }
+        self.change_respondent_details(payload, 200)
```

Comment:
Not keen on this approach of asserting. It's not very clear that change_respondent_details is doing the assertion by the name. I also think it's more readable if the assertion is done in the test method